### DISCUSSION TITLE
❤️值得拥有：Web3 URL 残酷共学第 1 期 徽章认领

### DISCUSSION THREAD

**Discussion by luffythink** [IntensiveCoLearning/Web3-URL#210] [Announcements]
参加「**Web3 URL 残酷共学 第 1 期**」并成功完成学习证明打卡的 LXer，将获得由 LXDAO 和 EthStorage 共同颁发的纪念徽章，共同见证这 21 DAYS 的残酷共学之旅。重点看图，看看都有哪些小伙伴？

![WICL1st](https://github.com/user-attachments/assets/c76847dd-6ed2-4d0e-a473-3aecfdbfe5b6)

在名单里的小伙伴记得在本贴下面登记一下自己的钱包地址， 运营的小伙伴会在本月给大家完成纪念徽章发放❤️

**Comment by linghuccc** [comment]
哪条链？需要钱包里面有 gas 么？

如果不需要的话

Ric Li C: 0xb72f70C0EA5462D39A26c635448ce2A0849d7f1E

**Reply by luffythink** [reply]
OP，不需要 gas 。

**Reply by linghuccc** [reply]
谢谢

**Comment by CHENFANGC** [comment]
Cora: 0xfd9daf85d7b50c0027cbdf5815f0450baf0a29ab

**Comment by nishuzumi** [comment]
Box:0x6E12A28086548B11dfcc20c75440E0B3c10721f5

**Comment by DerickIT** [comment]
Derick: 0x4c4F501eeC58bBD055D5D3dB61782d7c4CBa7a39

**Comment by punkcanyang** [comment]
Punkcan：0x1d41D6B1091C1a8A334096771bd1776019243d5e

**Comment by antigone4224** [comment]
Antigone4224（Irene）：
0x73c8fe6a2da25e5c9c654decc455e7f04ebe543e

**Comment by jjeejj** [comment]
Iyi: 0x2eC78D4677FBA1b5A589C66e75b0F6e180D8A66f

**Comment by YuKirasawa** [comment]
YuKirasawa: 0xdf33654b75055f14D0aF487EFc52744a0d89eF49

**Reply by wodeche** [reply]
可以麻烦去 LXDAO 官网注册 https://lxdao.io/buidlers 一下吗？或者加我wx ：wyx386553897

**Comment by zedz** [comment]
0xf95bdaAA2391ECedfD5cbb32B51C86360b0654F2

zedz
